_docs/2017-01-01-index.md

```diff
@@ -151,9 +151,11 @@ which is a worthy read for anyone with a background in cryptocurrency
 theory. The core idea of Proof of Stake is that instead of wasting
 electricity on cracking computationally heavy problems, a node is
 selected to mint a new block, with a probability proportional
-to the amount of coins a particular node has. A node that attempts to
-generate a block is called _a slotleader_. You can read more about
-this process in [Proof of Stake in Cardano SL](/cardano/proof-of-stake/).
+to the amount of coins a particular node has. Every node of network can attempt to
+generate a block. If node has positive [stake](/cardano/proof-of-stake/#stake) it
+is called _a stakeholder_. If node eventually becomes chosen to mint block
+it becomes _slotleader_. You can read more about this process in
```

Comment:
Well, being chosen to mint block and being a slotleader (btw, is it intentional that there is no space?) are the same things actually. This sentence is like `if you become chosen to rule a country you become a president`. I am not sure about usefulness of such sentence.